### ISSUE TITLE
Artifact not found in Maven Central

### ISSUE THREAD

**Issue by sanjarj** [placemarkt/address-formatter-java#2]
Hi,

I'm trying to inclure the lib as a dependecy using maven coordinates from readme, but getting:
```
Could not find artifact net.placemarkt:address-formatter-java:jar:0.0.9 in central (https://repo.maven.apache.org/maven2)
``` 

Is the artifact supposed to be found in maven central, or should I use some other repo?

**Comment by rkoeze** [comment]
Hi! You should use the Github repo. You'll need to add a repository to your `settings.xml` that looks something like this:

```
<repository>
     <id>github</id>
     <url>https://maven.pkg.github.com/OWNER/*</url>
</repository>
```

Let me know if it works. Check out [this Stack Overflow answer](https://stackoverflow.com/questions/58325279/mvn-install-package-from-github-package-registry) and [the Github Maven docs](https://docs.github.com/en/packages/guides/configuring-apache-maven-for-use-with-github-packages#authenticating-to-github-packages) for more information.

I'm also happy to mirror the package in Maven Central. I won't be able to get to it before this weekend though.

**Comment by sanjarj** [comment]
Mirroring the package to Maven Central would be the best, since pulling the dependency from GitHub Packages [requires authentication](https://github.community/t/how-to-allow-unauthorised-read-access-to-github-packages-maven-repository/115517/3).

**Comment by rkoeze** [comment]
Happy to do that then.

**Comment by chyoungie** [comment]
Is this still ongoing?

**Comment by rkoeze** [comment]
Hey! Sorry, life kinda caught up with me. I'll have this addressed this weekend.

**Comment by rkoeze** [comment]
Hey @chyoungie, just wanted to touch base. I've uploaded to the Sonatype Nexus repository manager and raised a ticket with them to sync to Maven Central. It looks like they have to do it manually, so it might be until Monday when it becomes available there. FWIW it is popping up in my search here already: https://s01.oss.sonatype.org/#nexus-search;quick~placemarkt

Closing this issue. Will update the Readme when Central syncs. Any more questions feel free to raise an issue. Thanks!
